README_ZH_CN.md

```diff
@@ -0,0 +1,111 @@
+# SigLens
+
+SigLens 是一个开源可观测性解决方案，比 Splunk 和 Elastic 高效 **100 倍**。
+
+## 为什么选择 SigLens：
+为 10,000 多名工程师提供可观测性工具的经验教会了我们一些事情：
+
+- 开发人员不得不在各种的工具之间切换以查看 logs、metrics 和 traces
+- Splunk、DataDog、NewRelic 的费用非常昂贵
+- ElasticSearch 需要太多的机器，集群维护比较困难
+- Grafana Loki 的查询性能较慢
+
+凭借几十年的监控领域经验，我们着手开始构建一个全新的，专为 logs、metrics 和 traces 而设计的，并且具有**零外部依赖**的可观测性数据库。只需一个**单一的二进制文件**，就可以在您的笔记本电脑上运行，处理每天高达 8 TB 的数据。
+
+## 加入我们的 Slack 社区
+在 [Slack](https://www.siglens.com/slack) 上和我们打个招呼吧 👋
+
```

Comment:
I guess you might need to add two line break characters `<br />` to align with the original document.